bindings/python/bt2/bt2/fields.py

```diff
@@ -366,6 +366,14 @@ def integer_field(self):
     def _set_value(self, value):
         self.integer_field.value = value
 
+    def __str__(self):
+        mapping_name = None
+        for mapping in self.mappings:
+            if self._value >= mapping.lower and self._value <= mapping.upper:
```

Comment:
Also, I'm thinking, since we can have any string as a label, use `repr()` for each label here (see my other comments for `_StructureField` and `_ArraySequenceField`, so that we have, for example:

    13 ('APPLE', "BAN'ANA", 'yes\'sir"miller')

What do you think?

Reply:
Good point about mappings() only returning matches anyway.
I'm fine with the representation you are proposing.